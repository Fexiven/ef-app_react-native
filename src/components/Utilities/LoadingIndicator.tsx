import { Text } from "react-native";

export const LoadingIndicator = () => {
    return <Text data-testid={"loadingindicator"}>We are loading some data, hang tight!</Text>;
};
