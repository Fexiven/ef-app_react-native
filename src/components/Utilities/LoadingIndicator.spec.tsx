import { render, screen } from "@testing-library/react-native";

import { LoadingIndicator } from "./LoadingIndicator";

describe("LoadingIndicator", function () {
    it("renders", () => {
        render(<LoadingIndicator />);

        expect(screen.getByTestId("loadingindicator")).toBeTruthy();
    });
});
