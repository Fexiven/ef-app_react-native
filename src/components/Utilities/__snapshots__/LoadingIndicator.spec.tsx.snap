// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator renders 1`] = `
<Text>
  We are loading some data, hang tight!
</Text>
`;
